XCTestBootstrap/TestManager/FBTestManagerTestReporterComposite.h

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBTestManagerTestReporter.h"
+#import <Foundation/Foundation.h>
+
```

Comment:
You should add `NS_ASSUME_NONNULL_BEGIN` `NS_ASSUME_NONNULL_END`, to expose the nullability contract in the type system